README.md

```diff
@@ -31,6 +31,9 @@ In addition, take note of the API Key value for your account. This value
 must be passed as an INORBIT_ICM_KEY environment variable to the
 service.
 
+Add INORBIT_BASE_URL environment variable with InOrbit base url. If you don't set up this variable
```

Comment:
:+1: for adding the comment.

Using a different server URL is a very advanced topic. I would suggest rephrasing it, flagging it as an advanced configuration and noting that if you are using a specialized InOrbit deployment then you may need to set this environment variable, but that for the vast majority of cases the default works well.